test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -4067,6 +4127,8 @@ static void traffic_mngr_test_tm_create(void)
 
 static void traffic_mngr_test_shaper(void)
 {
+	CU_ASSERT_FATAL(odp_tm_systems[0] != ODP_TM_INVALID);
```

Comment:
I do not quite see why we need these CUnit asserts in all these test functions. It seems a bit out of place as I think the CUnit assert should be where the TM system is created and the rest of the tests should not even run if things failed already at TM creation.